lib/ldp.js

```diff
@@ -538,6 +566,10 @@ class LDP {
     }
     return ensureNotExists(this, path.join(containerURI, filename))
   }
+
+  mimetypeIsRdf (mimeType) {
+    return RDF_MIME_TYPES.indexOf(mimeType) !== -1
```

Comment:
Assuming `RDF_MIME_TYPES` is an Array you can use [Array.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) I guess.

Reply:
Better, make RDF_MIME_TYPES a Set.

Reply:
> Assuming `RDF_MIME_TYPES` is an Array you can use [Array.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) I guess.

I actually did this initially when I wrote the code, but it failed for me, and I automatically reverted to Array.prototype.indexOf without much more research. My bad.

But yeah, will try making it a Set.